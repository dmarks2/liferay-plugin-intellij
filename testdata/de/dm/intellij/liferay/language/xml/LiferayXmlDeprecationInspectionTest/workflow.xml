<?xml version="1.0"?>

<warning descr="The XML Schema Location should be matched with the Liferay version"><workflow-definition
		xmlns="urn:liferay.com:liferay-workflow_7.0.0"
		xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
		xsi:schemaLocation="urn:liferay.com:liferay-workflow_7.0.0 http://www.liferay.com/dtd/liferay-workflow-definition_7_0_0.xsd"
>
	<name>My Workflow</name>
	<description>My Workflow</description>
	<version>1</version>
	<state>
		<name>created</name>
	</state>
</workflow-definition></warning>