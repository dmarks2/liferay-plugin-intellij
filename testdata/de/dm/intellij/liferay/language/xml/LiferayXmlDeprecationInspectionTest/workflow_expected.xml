<?xml version="1.0"?>

<workflow-definition xmlns="urn:liferay.com:liferay-workflow_7.4.0"
                     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                     xsi:schemaLocation="urn:liferay.com:liferay-workflow_7.4.0 http://www.liferay.com/dtd/liferay-workflow-definition_7_4_0.xsd">
    <name>My Workflow</name>
    <description>My Workflow</description>
    <version>1</version>
    <state>
        <name>created</name>
    </state>
</workflow-definition>